system/config/server.properties

```diff
@@ -584,4 +584,22 @@ MAX_SUMMARY_CACHE_SIZE=10000
 ###################################################
 disableHTMLCleaningExtensions=false
 
-allowLinksToOrphans=true
\ No newline at end of file
+allowLinksToOrphans=true
+
+###################################################
+# This command will be used for taking screenshot using headless browser
+# @file@ will be replaced with correct location of screenshot and
+# @url@ will be replaced with url of page for screenshot.
+# The path to executor needs to be replaced based on the browser that needs to be used.
+#
+# for Chrome command will be like:-
+#"C:/Program Files/Google/Chrome/Application/chrome.exe" --headless --disable-gpu --screenshot=@@file@@ @@url@@
+#
+# For Firefox command will be like:-
+#"C:/Program Files/Mozilla Firefox/firefox.exe" -headless -screenshot @@file@@ @@url@@
+#
+#For Edge command will be like:-
+#"C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe" -headless -screenshot=@@file@@ @@url@@
+#
+###################################################
+screenshotCommandLine=\"C\:Program Files/Mozilla Firefox/firefox.exe\" -headless -screenshot @@file@@ @@url@@
```

Comment:
@sonydeswal74  I think this will break new installs if they aren't on windows and don't have firefox installed.  We should default to blank and

Reply:
Oops Missed it. I fixed it.